GitCommands/Git/GitRevision.cs

```diff
@@ -86,6 +84,76 @@ public BuildInfo BuildStatus
         public string Subject { get; set; } = "";
         [CanBeNull]
         public string Body { get; set; }
+
+        [CanBeNull]
+        public string BodySummary
+        {
+            get
+            {
+                if (string.IsNullOrWhiteSpace(Body) || !HasMultiLineMessage)
+                {
+                    return Body;
```

Comment:
What would happen if `Body` is a single line but a very long line? Won't we need to trim it?